### ISSUE TITLE
Trained DALL-E result

### ISSUE THREAD

**Issue by adkuln** [lucidrains/DALLE-pytorch#18]
Hi, you showed good result of training.

I tried to use flickr-8k, 12 epochs, but result of inference is a pink background.. I can't understand why.

How many epochs and what the data do you use? 

Thanks

**Comment by lucidrains** [comment]
@adkuln Hi Aleksey! Is this 12 epochs for the VAE or for DALL-E? Can you give more details on your hyperparameters?

**Comment by adkuln** [comment]
> @adkuln Hi Aleksey! Is this 12 epochs for the VAE or for DALL-E? Can you give more details on your hyperparameters?

Hi! I trained VAE before the main part. VAE is okay i think. I trained with flickr-8k
After it, i trained DALL-E with flickr-8k also and result looks like this 
![image](https://user-images.githubusercontent.com/49467968/104898706-04282c80-598b-11eb-994c-78ba581597c3.png)

My params for training:
vae = DiscreteVAE(
    image_size = 256,
    num_layers = 3,
    num_tokens = 8192,
    codebook_dim = 256,
    hidden_dim = 128,
    temperature = 0.9
)

dalle = DALLE(
    dim = 256,
    vae = vae,                  # automatically infer (1) image sequence length and (2) number of image tokens
    num_text_tokens = 100000,    # vocab size for text
    text_seq_len = 128,         # text sequence length
    depth = 6,                  # should be 64
    heads = 8,                  # attention heads
    dim_head = 64,              # attention head dimension
    attn_dropout = 0.1,         # attention dropout
    ff_dropout = 0.1            # feedforward dropout
)

10-12 epochs

**Comment by adkuln** [comment]
its worked good. Thanks. I found bug. 

But, also i checked that it doesn't work with multi-gpu. I tried with PL training. Have you checked this functional?

**Comment by powderblock** [comment]
> its worked good. Thanks. I found bug.
> 
> But, also i checked that it doesn't work with multi-gpu. I tried with PL training. Have you checked this functional?

Hey, what was the bug you found? 👍

**Comment by johndpope** [comment]
close
